hail_search/hail_search_query.py

```diff
@@ -203,13 +204,25 @@ def import_filtered_table(self, data_type, sample_data, **kwargs):
 
         return ht
 
-    def _filter_entries_table(self, ht, sample_data, inheritance_mode=None, inheritance_filter=None, **kwargs):
+    def _filter_entries_table(self, ht, sample_data, inheritance_mode=None, inheritance_filter=None, quality_filter=None,
+                              **kwargs):
         ht, sample_id_family_index_map = self._add_entry_sample_families(ht, sample_data)
 
         ht = self._filter_inheritance(
             ht, inheritance_mode, inheritance_filter, sample_data, sample_id_family_index_map,
         )
 
+        quality_filter = quality_filter or {}
+        if quality_filter.get('vcf_filter'):
```

Comment:
It seems to me that the `get` function can be `pop`. Otherwise, the `vcf_filter` field will be an unnecessary round of iterating in `_get_genotype_passes_quality_filter`.

Reply:
the additional iterating is necessary regardless, because there may be quality filters that are data type specific to a different data type. So for instance, if there is an SV-specific quality filter we do not want to apply it here. And then since we are iterating anyways, theres no need to modify the input unnecessarily